### PR TITLE
U/fjammes/dm 3104

### PR DESCRIPTION
**Pull request by jbecla**

### CODE REVIEW

File: datasets/case03/queries/0019.2_selectRunDeepSourceDeepcoaddDeepsrcmatchRefobject.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+-- Derived from https://dev.lsstcorp.org/trac/wiki/dbPipeQAQueries
+-- Used to test BIT field support
+-- Here ORDER BY fails for unknow reason
+
+SELECT sce.filterName, sce.tract, sce.patch, sro.gMag, sro.ra, sro.decl, sro.isStar,
+       sro.refObjectId, s.id as sourceId,  rom.nSrcMatches, s.flags_pixel_interpolated_center, s.flags_negative,
```
Comment (jbecla) [inline datasets/case03/queries/0019.2_selectRunDeepSourceDeepcoaddDeepsrcmatchRefobject.sql]: remove extra space, maybe make the line a bit shorter (move last element to next line)

File: bin/qserv-test-head.sh [hunk 2]
```diff
@@ -35,10 +35,10 @@ Usage: `basename $0` [options]
...
-    -q          quick: only rebuilds and install new Qserv code
-                       instead of rebuilding from scratch,
-                       and performs test case #01 instead of all
-                       integration tests.
+    -r          rebuild: clean an rebuild whole Qserv code
+    -C          configure: configure Qserv mono-node instance
+    -c          Specify which test case number to run, all tests
+                are run by default
     -R          Specify Qserv execution directory, default to
                 \$HOME/qserv-run/git
```
Comment (andy-slac) [inline bin/qserv-test-head.sh]: git?

File: datasets/case01/queries/0012.1_raftAndCcd.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+-- interesting syntax
```
Comment (andy-slac) [inline datasets/case01/queries/0012.1_raftAndCcd.sql]: Is this comment useful?

File: datasets/case01/queries/0012.2_raftAndCcd.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+-- interesting syntax
```
Comment (andy-slac) [inline datasets/case01/queries/0012.2_raftAndCcd.sql]: same here

File: datasets/case01/queries/1004.1_varObjects.sql [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+-- AVG() still has to trigger aggregation
+-- step. 
+
+-- pragma noheader
+SELECT AVG(ra_PS) as ra
+FROM   Object
+WHERE  ra_PS BETWEEN 0 AND 3   -- noQserv
+  AND  decl_PS BETWEEN 0 AND 10 -- noQserv
+-- withQserv WHERE qserv_areaspec_box(0, 0, 3, 10)
+--   AND variability > 0.8
```
Comment (andy-slac) [inline datasets/case01/queries/1004.1_varObjects.sql]: This line is completely commented out, either remove or add comment why

File: bin/qserv-test-head.sh [hunk 6]
```diff
@@ -35,10 +35,10 @@ Usage: `basename $0` [options]
 
   Available options:
     -h          this message
-    -q          quick: only rebuilds and install new Qserv code
-                       instead of rebuilding from scratch,
-                       and performs test case #01 instead of all
-                       integration tests.
+    -r          rebuild: clean an rebuild whole Qserv code
```
Comment (andy-slac) [inline bin/qserv-test-head.sh]: and instead of an?

**Pull request merged**